knowledge_repo/app/routes/comment.py

```diff
@@ -35,9 +35,13 @@ def post_comment():
     if not post:
         raise Exception('Unable to find post')
 
-    comment = (db_session.query(Comment)
-                         .filter(Comment.id == comment_id)
-                         .first())
+    if comment_id is not None:
+        comment = (db_session.query(Comment)
```

Comment:
I believe this is equivalent to `if comment_id`, since `comment_id` will either be an integer or `None`